### PR TITLE
Wip/superm1/simmer down

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/hughsie/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-udev-device.c [hunk 1]
```diff
@@ -194,6 +194,8 @@ fu_udev_device_probe (FuDevice *device, GError **error)
 			tmp = g_udev_device_get_property (priv->udev_device, "ID_MODEL_FROM_DATABASE");
 		if (tmp == NULL)
 			tmp = g_udev_device_get_property (priv->udev_device, "ID_MODEL");
+		if (tmp == NULL)
+			tmp = g_udev_device_get_property (priv->udev_device, "ID_PCI_CLASS_FROM_DATABASE");
```
Comment (hughsie) [inline src/fu-udev-device.c]: I guess it's not ideal, but at this point we're kinda clutching at straws.

File: src/fu-plugin.c [hunk 2]
```diff
@@ -1555,8 +1555,14 @@ fu_plugin_runner_udev_device_added (FuPlugin *self, FuUdevDevice *device, GError
 	if (func == NULL) {
 		if (priv->device_gtype != G_TYPE_INVALID ||
 		    fu_device_get_specialized_gtype (FU_DEVICE (device)) != G_TYPE_INVALID) {
-			g_debug ("using generic udev_device_added() on %s", priv->name);
-			return fu_plugin_udev_device_added (self, device, error);
+			if (!fu_plugin_udev_device_added (self, device, &error_local)) {
+				if (g_error_matches (error_local,
+						     FWUPD_ERROR,
+						     FWUPD_ERROR_NOT_SUPPORTED))
+					return TRUE;
```
Comment (hughsie) [inline src/fu-plugin.c]: maybe we want the same in fu_plugin_runner_usb_device_added?

File: src/fu-udev-device.c (hunk 1 above)
Comment (superm1) [reply src/fu-udev-device.c]: I was thinking the same, but consider what we're doing for synaptics touchpad.  Then at least it's classed right.

File: src/fu-plugin.c (hunk 2 above)
Comment (superm1) [reply src/fu-plugin.c]: Yes, just added.

**Pull request merged**